### PR TITLE
Adding faq about underline order

### PR DESCRIPTION
**Pull request by Grummfy**

### CODE REVIEW

**Comment by marmotz** [comment]
In fact, I think you should move the top comments of index.rst in this file

**Comment by marmotz** [comment]
@Grummfy ? Have you seen my comment ?

**Comment by Grummfy** [comment]
I'm a bit too busy right now.

**Comment by Grummfy** [comment]
here we go

File: README.md [hunk 1]
```diff
@@ -62,4 +62,5 @@ They must stay the same. It will be easier for other people to translate other p
 ### Where can I find the syntax for the files?
 We use the reStructuredText syntax. You can find it on the [sphinx website](http://sphinx-doc.org/rest.html).
 
-
+### What are the title underline order?
+In reStructuredText synthaxe, you can use a wide variety of character to underline. The order use in the atoum documentation is : #, *, =, -, ", ^, `, :, ., ' . Now if you have more than 4 subdivision, think subdivide into multiple files.
```
Comment (marmotz) [inline README.md]: syntaxe, subdivisions

**Comment by Grummfy** [comment]
happy you are more conscientious than me! ;)

**Pull request merged**